### PR TITLE
fix spotlesssql errors

### LINKED ISSUE #24988: spotlessSql check fails in source connectors
Steps to reproduce. Run from airbyte root:
`./gradlew spotlessSql`

**Expected**:
Command completes without any errors.

**Actual**:
```
> Task :spotlessSql FAILED
Step 'dbeaverSql' found problem in 'airbyte-integrations/connectors/source-mysql/integration_tests/seed/full.sql':
index 27370, length 27370
```

### PR DESCRIPTION
**Pull request by rodireich**
## What
Spotlesssql failed on two files:
* `…/source-mysql/integration_tests/seed/full.sql` for an escaped single quote `\'`
* `…/source-oracle/integration_tests/seed/full.sql` for an unescaped single quote (legitimate error)

closes #24988.

## How
Switched escaping of single quote to the other "classic" way - two single quotes `''` which is equivalent and accepted by spotless.

### CODE REVIEW

File: airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json [hunk 1]
```diff
@@ -15382,7 +15382,7 @@
     "sourceDefinitionId": "38f84314-fe6a-4257-97be-a8dcd942d693",
     "name": "Insightly",
     "dockerRepository": "airbyte/source-insightly",
-    "dockerImageTag": "0.1.1",
```
Comment (erohmensing) [inline airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json:15385]: Merging master in should fix these automated changes

**Review by erohmensing — APPROVED** [review]
Thank you thank you!

File: airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json (hunk 1 above)
Comment (erohmensing) [reply airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json:15385]: Oh actually I'm wrong, looks like they weren't updated correctly in the other PR. in any case I think these are valid chagnes

Comment (rodireich) [reply airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json:15385]: I know a trick to get around this issue of unable to merge

**Comment by octavia-approvington** [comment]
Our work here is done <br/> done

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**